### PR TITLE
Task/updates radio component

### PR DESCRIPTION
**Pull request by clayton-duarte**
Updates the radio component so it renders the right number of columns.

### CODE REVIEW

File: src/components/Radio/index.tsx [hunk 1]
```diff
@@ -53,8 +54,21 @@ const Radio: FunctionComponent<RadioProps & HTMLAttributes<HTMLDivElement>> = ({
...
+  const getColumnCount = (): number => {
+    switch (options.length) {
+      case 0:
+      case 1:
+        return 1
+      case 2:
+      case 4:
+        return 2
+      default:
+        return 3
```
Comment (l0rdcafe) [inline src/components/Radio/index.tsx:66]: question: any reason why not to return 0 when case 0? also have you tested with weird or bigger numbers?

**Review by l0rdcafe — APPROVED** [review]

File: src/components/Radio/index.tsx [hunk 2]
```diff
@@ -33,9 +33,10 @@ const Radio: FunctionComponent<RadioProps & HTMLAttributes<HTMLDivElement>> = ({
   ...containerProps
 }) => {
   const [value, setValue] = useState<string>('')
+  console.log(options)
```
Comment (pajanpl) [inline src/components/Radio/index.tsx:36]: Is this just for debugging?

**Review by pajanpl — DISMISSED** [review]
👍

File: src/components/Radio/index.tsx (hunk 2 above)
Comment (clayton-duarte) [reply src/components/Radio/index.tsx:36]: whoops, removing that

Comment (pajanpl) [reply src/components/Radio/index.tsx]: I also noticed that the alerts just say `[object Object]`, is that alright?

**Review by pajanpl — APPROVED** [review]

**Pull request merged**